src/sentry/utils/email/backend.py

```diff
@@ -39,14 +39,10 @@ class PreviewBackend(BaseEmailBackend):  # type: ignore
     def send_messages(self, email_messages: Sequence[EmailMultiAlternatives]) -> int:
         for message in email_messages:
             content = bytes(message.message())
-            preview = tempfile.NamedTemporaryFile(
+            with tempfile.NamedTemporaryFile(
                 delete=False, prefix="sentry-email-preview-", suffix=".eml"
-            )
-            try:
+            ) as preview:
                 preview.write(content)
-                preview.flush()
-            finally:
-                preview.close()
```

Comment:
Was the flush and close not needed?

Reply:
the `with` statement does both of those things for us!